tests/test_gaap.py

```diff
@@ -0,0 +1,183 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+from __future__ import absolute_import, division, print_function
+
+import math
+import unittest
+import galsim
+import lsst.utils.tests
+import lsst.daf.base as dafBase
+import lsst.afw.detection as afwDetection
+import lsst.afw.geom as afwGeom
+import lsst.geom as geom
+import lsst.afw.table as afwTable
+import lsst.afw.image as afwImage
+import lsst.meas.base as measBase
+import lsst.afw.display as afwDisplay
+import lsst.meas.extensions.gaap
+
+
+try:
+    type(display)
+except NameError:
+    display = False
+    frame = 1
+
+
+def getGaapResultName(sF: float, sigma: float, name: str) -> str:
+    return "_".join((name, str(sF).replace(".", "_")+"x", str(sigma).replace(".", "_")))
+
+
+def makeGalaxyExposure(bbox, scale, psfSigma=0.9, flux=1000., galSigma=3.7):
+    psfWidth = 2*int(4.0*psfSigma) + 1
+    galWidth = 2*int(40.*math.hypot(galSigma, psfSigma)) + 1
+    gal = galsim.Gaussian(sigma=galSigma, flux=flux)
+
+    galIm = galsim.Image(galWidth, galWidth)
+    galIm = galsim.Convolve([gal, galsim.Gaussian(sigma=psfSigma, flux=1.)]).drawImage(image=galIm,
+                                                                                       scale=1.0,
+                                                                                       method='no_pixel')
+    exposure = afwImage.makeExposure(afwImage.makeMaskedImageFromArrays(galIm.array))
+    exposure.setPsf(afwDetection.GaussianPsf(psfWidth, psfWidth, psfSigma))
+
+    exposure.getMaskedImage().getVariance().set(1.0)
+    exposure.getMaskedImage().getMask().set(0)
+    center = exposure.getBBox().getCenter()
+
+    cdMatrix = afwGeom.makeCdMatrix(scale=scale)
+    exposure.setWcs(afwGeom.makeSkyWcs(crpix=center,
+                                       crval=geom.SpherePoint(0.0, 0.0, geom.degrees),
+                                       cdMatrix=cdMatrix))
+    return exposure, center
+
+
+class GaapFluxTestCase(lsst.utils.tests.TestCase):
```

Comment:
Should have a docstring, even if it's a very brief one.  Same for all test methods: briefly summarize what each test is testing (and what it isn't, if that's important and non-obvious).